client/components/Catalog/Catalog.scss

```diff
@@ -0,0 +1,11 @@
+.catalog {
+  z-index: 10;
+  // width: 115%;
+  width: 100%;
+  padding: 40px 25px 0;
+  max-width: 1200px;
+
+  @media (max-width: 768px) {
+    width: 100%;
```

Comment:
@AStashevskaya Looks like this media query is optional, becaue `width` is always `100%`